recipes/r-rcppparallel/meta.yaml

```diff
@@ -13,48 +8,27 @@ source:
     - http://cran.r-project.org/src/contrib/RcppParallel_4.3.20.tar.gz
     - http://cran.r-project.org/src/contrib/Archive/RcppParallel/RcppParallel_4.3.20.tar.gz
 
-
-  # You can add a hash for the file here, like md5 or sha1
-  # md5: 49448ba4863157652311cc5ea4fea3ea
-  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
-  # patches:
-   # List any patch files here
-   # - fix.patch
-
 build:
-  # If this is a new build for the same version, increment the build
-  # number. If you do not include this key, it defaults to 0.
-  # number: 1
-
-  # This is required to make R link correctly on Linux.
+  number: 0
```

Comment:
One last thing, you need to increase the version number, otherwise nothing will be tested.